ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/FollowerPromptness.hs

```diff
@@ -0,0 +1,214 @@
+{-# LANGUAGE BlockArguments      #-}
```

Comment:
Just personal grumbling about `BlockArguments` and `TupleSections`. I haven't lobbied to ban these in the style guide, but I consider them more trouble than they're worth. (I'll edit this comment if I change my mind by the end of the module 😄)

Reply:
I only have extremely weak opinions here; I personally see no downside (but also no big benefit) in being to avoid extraneous `$`s/parens/lambda abstractions and just immediately enable them when I hit such a case.